package.json

```diff
@@ -29,10 +29,10 @@
     "build:src": "webpack --config webpack-main.config.js",
     "build:test": "webpack --config webpack-test.config.js",
     "build": "npm run build:src && npm run build:test",
-    "lint:js": "eslint src/main/webapp/dev -c static-analysis/teammates-eslint.yml --ext es6",
+    "lint:js": "eslint *.js src/main/webapp/dev -c static-analysis/teammates-eslint.yml --ext js",
     "lint:json": "eslint src/main/webapp/data/*.json src/main/resources/*.json src/test/resources/data/*.json -c static-analysis/teammates-eslint-json.yml",
     "lint:css": "stylelint src/main/webapp/stylesheets/*.css --config static-analysis/teammates-stylelint.yml",
-    "lint:spaces": "lintspaces -n -t -d spaces -l 1 -. \"src/**/*.jsp\" \"src/**/*.tag\" \"src/main/**/*.html\" \"src/**/*.xml\" \"src/**/*.json\" \"src/**/*.css\" \"src/**/*.java\" \"src/**/*.es6\" \"src/**/*.properties\" \"*.yml\" \"*.gradle\" \"static-analysis/*.*ml\"",
+    "lint:spaces": "lintspaces -n -t -d spaces -l 1 -. \"src/**/*.jsp\" \"src/**/*.tag\" \"src/main/**/*.html\" \"src/**/*.xml\" \"src/**/*.json\" \"src/**/*.css\" \"src/**/*.java\" \"src/main/webapp/dev/**/*.js\" \"src/**/*.properties\" \"*.js\" \"*.yml\" \"*.gradle\" \"static-analysis/*.*ml\"",
```

Comment:
Not completely related but it seems we missed out linting the Webpack config files previously right?

Is there any way this can be less error prone?

Reply:
You can't say this is "error prone". It is incomplete, but everything works as intended.

Reply:
Sorry, I don't really get what you mean. What I meant is that it can be easy to miss some files, for example, we missed out linting the Webpack config files.

So I am asking if we could actually avoid making such errors, maybe for example by writing tests?

Reply:
Why would you write test for linters? That's an unnecessary resource wasted and later on maintenance burden for quite a marginal gain.

Missing linting the Webpack config files is unfortunate, but does not really affect us in any way.

Reply:
I guess that is a bad example, I was hoping that there is a simple way to make sure we don't miss linting any files. I suppose that's not possible then.

Reply:
That is not possible for one very basic fact that each linter works and is configured differently.